kvui.py

```diff
@@ -320,31 +387,30 @@ def __init__(self):
         self.entrance_text = ""
         self.status_text = ""
         self.hint = {}
-        for child in self.children:
-            child.bind(texture_size=self.set_height)
 
+        ctx = MDApp.get_running_app().ctx
+        menu_items = []
 
-        ctx = App.get_running_app().ctx
-        self.dropdown = DropDown()
+        for status in (HintStatus.HINT_NO_PRIORITY, HintStatus.HINT_PRIORITY, HintStatus.HINT_AVOID):
+            name = status_names[status]
+            status_button = MDDropDownItem(MDDropDownItemText(text=name), size_hint_y=None, height=dp(50))
+            status_button.status = status
+            menu_items.append({
+                "text": name,
+                "leading_icon": status_icons[status],
+                "on_release": lambda x=status: select(self, x)
+            })
 
-        def set_value(button):
-            self.dropdown.select(button.status)
+        self.dropdown = MDDropdownMenu(caller=self.ids["status"], items=menu_items)
 
         def select(instance, data):
             ctx.update_hint(self.hint["location"],
                             self.hint["finding_player"],
                             data)
 
-        for status in (HintStatus.HINT_NO_PRIORITY, HintStatus.HINT_PRIORITY, HintStatus.HINT_AVOID):
-            name = status_names[status]
-            status_button = Button(text=name, size_hint_y=None, height=dp(50))
-            status_button.status = status
-            status_button.bind(on_release=set_value)
-            self.dropdown.add_widget(status_button)
-
-        self.dropdown.bind(on_select=select)
+        self.dropdown.bind(on_release=self.dropdown.dismiss)
 
-    def set_height(self, instance, value):
+    def set_height(self, _, _2):
```

Comment:
In Python, using underscores for unused function parameter names is not good practice.
It's good for local variables, but not for function parameters.

In Python (different from some other languages), the parameter names are an important part of the function signature.
If this is an override or if it's implementing some interface, and the interface is defined as:
```python
def set_height(self, instance, value):
    ...
```
That means that someone could call an implementor of that interface with:
```python
x.set_height(instance=foo, value=bar)
```
And that would crash with this override, because it doesn't see those parameter names.

If a linter is complaining about these unused parameters, it shouldn't,
because it can't know whether you're implementing some structural interface that requires the parameters (with certain names) to be there.